packages/calypso-e2e/package.json

```diff
@@ -40,7 +40,6 @@
 	},
 	"scripts": {
 		"clean": "yarn build --clean && rm -rf dist",
-		"build": "tsc --build ./tsconfig.json",
-		"test": "yarn jest"
```

Comment:
This doesn't work, and no other packages try to do this. The monorepo is set up with the intention of running unit tests from the top level of the repo.